### PR TITLE
add port options for wordpress mysql db connection

### PR DESCRIPTION
**Pull request by passos**

### CODE REVIEW

**Comment by mattr-** [comment]
@jekyllbot: merge

File: lib/jekyll-import/importers/wordpress.rb [hunk 1]
```diff
@@ -77,6 +79,7 @@ def self.process(opts)
           :user           => opts.fetch('user', ''),
           :pass           => opts.fetch('password', ''),
           :host           => opts.fetch('host', 'localhost'),
+          :port           => opts.fetch('port', '3306'),
```
Comment (parkr) [inline lib/jekyll-import/importers/wordpress.rb]: Should this be a string 3306, or an integer 3306?

**Comment by parkr** [comment]
The documentation also needs to be updated.

File: lib/jekyll-import/importers/wordpress.rb (hunk 1 above)
Comment (mattr-) [inline lib/jekyll-import/importers/wordpress.rb]: The `mysql2` gem will handle both an integer and a string so it doesn't matter.

**Pull request merged**